include/swift/AST/Types.h

```diff
@@ -2713,7 +2711,7 @@ enum class ParameterConvention {
   Direct_Guaranteed,
 };
 
-inline bool isIndirectParameter(ParameterConvention conv) {
+inline bool isIndirectFormalParameter(ParameterConvention conv) {
```

Comment:
Sure. Despite the appearance, I was trying *not* to do unnecessary cleanup in this PR.